dev/loadgen/cmd/benchmark.go

```diff
@@ -55,7 +55,7 @@ var benchmarkCommand = &cobra.Command{
 		}
 
 		var load loadgen.LoadGenerator
-		load = loadgen.NewFixedLoadGenerator(500*time.Millisecond, 300*time.Millisecond)
+		load = loadgen.NewFixedLoadGenerator(800*time.Millisecond, 300*time.Millisecond)
```

Comment:
slightly decreased the rate, this was creating workspaces too quickly for mk2, as mk2's StartWorkspace request doesn't block and was following the 2/second rate.

For mk1 loadtests, the StartWorkspace request takes seconds (to minutes) to complete, and never reached a rate of 2 starts/second anyways.

Reply:
> slightly decreased the rate, this was creating workspaces too quickly for mk2, as mk2's StartWorkspace request doesn't block and was following the 2/second rate.

@WVerlaek were you hitting a rate limit of ws-manager-mk2, where it wasn't allowing additional gRPC connections? I assume yes, just curious.

Reply:
no, some workspaces were failing to start, there were too many starting at once and pulling an image causing some to fail to pull. Increased the delay a bit to slow down workspace creation, but at this rate it's still faster than what mk1 would handle.

Reply:
Cool, I see, so it's just a natural breaking limit. Good to know! 

What was failing on pull? registry-facade, containerd? Something else? Just curious.

Reply:
The errors were failing to pull image from registry-facade due to IO timeout